src/static/riot/tasks/management.tag

```diff
@@ -64,28 +106,63 @@
          Init
         ---------------------------------------------------------------------*/
 
-        self.tasks = []
+        self.my_tasks = []
+        self.public_tasks = []
 
         self.one("mount", function () {
-
             self.update_tasks()
-
+            $('.tabular.menu .item', self.root).tab()
         })
 
         self.update_tasks = function () {
-            CODALAB.api.get_tasks()
+            self.update_my_tasks()
+            self.update_public_tasks()
+        }
+
+        self.update_my_tasks = function (filters) {
+            filters = filters || {}
+            filters.created_by = CODALAB.state.user.id
+            CODALAB.api.get_tasks(filters)
                 .done(function (data) {
-                    self.tasks = data
+                    self.my_tasks = data
                     self.update()
                 })
                 .fail(function (response) {
                     toastr.error("Could not load tasks")
                 })
         }
 
+        self.update_public_tasks = function (filters) {
+            filters = filters || {}
+            filters.is_public = true
+            CODALAB.api.get_tasks(filters)
+                .done(function (data) {
+                    self.public_tasks = data
+                    self.update()
+                })
+                .fail(function (response) {
+                    toastr.error("Could not load tasks")
+                })
+        }
+
+
+        self.search_my_tasks = function () {
+            var filter = self.refs.search_mine.value
+
+            if (filter !== "" && filter.length < 3) {
+
```

Comment:
can just return here and then we don't need the `else` code indented? also, could maybe reduce the delay code a bit...

```delay(() => self.update_my_tasks({search: filter}), 100)```